tests/framework/extensions/projects/projects.go

```diff
@@ -32,3 +32,33 @@ func GetProjectByName(client *rancher.Client, clusterID, projectName string) (*m
 
 	return project, nil
 }
+
+// GetProjectList is a helper function that returns all the project in a specific cluster
+func GetProjectList(client *rancher.Client, clusterID string) (*management.ProjectCollection, error) {
+	var project *management.ProjectCollection
+
+	projectsList, err := client.Management.Project.List(&types.ListOpts{
+		Filters: map[string]interface{}{
+			"clusterId": clusterID,
+		},
+	})
+	if err != nil {
+		return project, err
+	}
+
+
+	return projectsList, nil
+}
+
+
+// GetProjectList is a helper function that deletes a project in a specific cluster
+func DeleteProject(client *rancher.Client, project *management.Project) (error) {
```

Comment:
Do you think we really need a helper function to delete a project? I don't think it's needed. a delete call is pretty simple to use